shuup/xtheme/template_ns.py

```diff
@@ -62,6 +62,11 @@ def get(self, context, name, default=None):
             return theme.get_setting(name, default=default)
         return default
 
+    @contextfunction
+    def get_current_language(self, context):
```

Comment:
Also here you should have language code in context already.

Reply:
I guess this also ok then?

Reply:
I removed this and inserted a `with context` when including this template. Not being a macro helps.